src/main/app/src/app/admin/model/formulier-definitie-verwijzing.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+export class FormulierDefinitieVerwijzing {
```

Comment:
In Flowable wordt de term Definition gebruikt i.p.v. Definitie. Ik stel voor dit FormDefinitionXXX te noemen

Reply:
dit is een verwijzing naar onze eigen formulieren, aanvullende informatie, advies welke formulier extenden, deze laat ik even zo